core/src/View/Kokkos_ViewCtor.hpp

```diff
@@ -113,7 +113,7 @@ struct ViewCtorProp<std::enable_if_t<is_view_label<Label>::value>, Label> {
   using type = std::string;
 
   ViewCtorProp(const type &arg) : value(arg) {}
-  ViewCtorProp(type &&arg) : value(arg) {}
+  ViewCtorProp(type &&arg) : value(std::move(arg)) {}
```

Comment:
Another possibility is to replace these two constructors with
```c++
ViewCtorProp(type arg) : value(std::move(arg)) {}
```

I'm not sure which is better though...

Reply:
leave it like it is